.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+Metrics/BlockLength:
+  Exclude:
+    - fastlane/Fastfile
+
+Style/AsciiComments:
+  Exclude:
+    - fastlane/Fastfile
+
+Layout/LineLength:
+  Exclude:
+    - fastlane/Fastfile
+
+Naming/FileName:
+  Exclude:
+    - fastlane/Matchfile
```

Comment:
I can only guess that `Fastfile` is already excluded by the rule, while `Matchfile` isn't. 

I'd love to look into it on the Rubocop end and see if I can address it.

Reply:
IMHO we should just get rid of the `Matchfile` anyway, so that would also be a solution 😁

Reply:
+1.

Maybe something I'll get to do over the next few weeks. Together with folding the `Deliverfile` and standalone `deliver` call into a lane to make it consistent with the rest of the workflow.